MANIFEST.in

```diff
@@ -1,27 +1,19 @@
-include MANIFEST.in
-include Makefile
 include common.sh
-include README*
 include LICENSE
-include preupg premigrate
-include preupg.bash
-include preupg-content-creator
+include MANIFEST.in
 
-recursive-include common *
-recursive-include kickstart *
-recursive-include man *
-recursive-include preuputils *
-recursive-include postupgrade.d *
-recursive-include scripts *.sh
+include           bin/*
+recursive-include data *
+recursive-include doc *
+recursive-include etc *
+include           man/*
+include           preupg/*.py
```

Comment:
The code will be changed in redhat-upgrade-tool, there is no problem with that. Correct dependency must be specified in the r-u-t spec file.
https://github.com/upgrades-migrations/redhat-upgrade-tool/issues/15